cmd/zetaclientd/debug.go

```diff
@@ -45,7 +45,7 @@ func init() {
 
 func DebugCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "get-ballot-from-intx [txHash] [chainID]",
+		Use:   "get-ballot-from-inbound [txHash] [chainID]",
```

Comment:
Would `get-ballot-for-inbound` or `get-inbound-ballot` be a better name?

Reply:
I think we can use `get-inbound-ballot`

Reply:
https://github.com/zeta-chain/node/pull/2118/commits/000e28ea285b6eeed10ef675769d1f2b32bb9dd7